src/main/scala/is/hail/variant/Variant.scala

```diff
@@ -46,6 +47,16 @@ object CopyState extends Enumeration {
 }
 
 object AltAllele {
+
+  def fromMemoryBuffer(m: MemoryBuffer, offset: Long): AltAllele = {
+    val t = TAltAllele.representation
+    val ref = TString.loadString(m, t.loadField(m, offset, 0))
+    val alt = TString.loadString(m, t.loadField(m, offset, 1))
+    AltAllele(ref, alt)
+  }
+
+  def fromRegionValue(rv: RegionValue, offset: Long): AltAllele = fromMemoryBuffer(rv.region, offset + rv.offset)
```

Comment:
This can't be right: take a `RegionValue` or a `MemoryBuffer` and `Long`, but not both.  Adding offset to a region value offset makes no sense.